src/app/quizPage.js

```diff
@@ -21,86 +26,50 @@ let GAME_HANDLER = null;
 export function renderQuizPage(mode, name, totalTime) {
     CURRENT_MODE = mode;
     GAME_HANDLER = new GameHandler(name, totalTime);
-
+    GENERATOR = new QuestionGenerator(CURRENT_MODE)
     const appScreen = document.querySelector('#pokequiz-app');
     appScreen.classList.add(QUIZ_PAGE_STYLES.quizPageClass)
     appScreen.classList.remove(START_PAGE_STYLES.startPageClass)
     const quizTemplate = document.getElementById('quiz-template');
     appScreen.innerHTML = quizTemplate.innerHTML;
-    // TODO later  - generate question using questionService - below are temporary dummy variables
-    const generatedQuestion = {
-        question: "quizQuestion",
-        questionNum: 1,
-    }
+
     setupPageTitle(CURRENT_MODE);
-    //GENERATOR = new QuestionService.Generator()
+    // = new QuestionService.Generator()
```

Comment:
removable comment.

Actually, commenting code too much is anti-pattern. It might help at the start, but comments should not be pushed for others. 
There are some why commenting is bad:
1. If you need to comment something, then the code is not readable enough. (Instead: try to encapsulate some logic in functions that have meaningful name)
2. Comments can easily expire and start to be invalid - when somebody changes your code he also needs to change comments, imagine he doesn't change them.
 a) The best for code with comments is that, when somebody updates code and comments.
 b) Good for code with comments is that, when somebody updates code and removes comments
 c) The worst for code with comments is that, when somebody **updates code** and **doesn't update comments**. It creates discrepancy - what is source of truth? Code? Or comment? What if somebody trusts the invalid comment instead of read the code? Bad things can happen and lots of time can be wasted